setup.py

```diff
@@ -2,18 +2,16 @@
 from os import path
 from io import open
 
-import versioneer
-
 here = path.abspath(path.dirname(__file__))
 
 # Get the long description from the README file
 with open(path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
 setup(name='pynvml',
-      version=versioneer.get_version(),
-      cmdclass=versioneer.get_cmdclass(),
+      version='12.0.0',
       python_requires='>=3.6',
```

Comment:
Is this still true?

Reply:
Hm, not actually sure.

Reply:
Am not seeing it specified in their `sdist`